client/routes.js

```diff
@@ -1,25 +1,18 @@
+// react lib imports
 import React from 'react';
-import { Route, IndexRoute } from 'react-router';
+import { Route, Switch } from 'react-router-dom';
 
 // components
-import App from './components/App';
-import LoginPage from './components/HomePage/LoginPage';
-import HomePage from './components/HomePage/HomePage';
-import EventPage from './components/Events/index';
-import SocialClubPage from './components/SocialClub/SocialClubPage';
-import EnsureLoggedIn from "./components/common/EnsureLoggedIn";
-import NotFound from "./components/common/NotFound";
+import Login from './pages/Login/Login';
+import NotFound from './components/common/NotFound';
 
+export const Routes = () => (
+  <div>
```

Comment:
Hey @williamolojede good job cleaning up the code, I wonder why you chose to use this `div` tag, it will add extra dom elements that doesn't really improve our style. Have you considered using something like [React Fragments](https://reactjs.org/docs/fragments.html).

Reply:
I'm not sure if you was supposed to cater for responsiveness but the page tends to bring a white container when viewed on smaller devices

Reply:
Thanks for spotting this out, I will cater for responsiveness